site/docs/tools/spark_auth_config.md

```diff
@@ -0,0 +1,55 @@
+# Authentication in Spark
+
+When Nessie is used in Spark-based environments (either with [Iceberg](./iceberg/index.md) 
+or [Delta Lake](./deltalake/index.md)) the Nessie authentication settings are configured via Spark session properties.
+
+=== "Java"
```

Comment:
copy-pasted from someplace else... how is markdown broken? it renders well to HTML via `mkdocs` in my env :thinking:

Reply:
you can "View File" directly here on the PR and it will show you the parsed markdown. If you replace === with ### then it will work on the website and via markdown :)

Reply:
we should probably update the other places that use "="

Reply:
no, it's not the same `===` generate a tabbed pane with example for different languages, e.g.: https://projectnessie.org/tools/iceberg/spark/

`###` is gives two sections one below the other.

Reply:
This is apparenetly handled by an extension that we use in our docs: https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/

Reply:
ah you're absolutely right. I just realized the same while I tried to update those